.baseline/checkstyle/custom-suppressions.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<!DOCTYPE suppressions PUBLIC
+    "-//Checkstyle//DTD SuppressionFilter Configuration 1.2//EN"
+    "https://checkstyle.org/dtds/suppressions_1_2.dtd">
+<!-- IMPORTANT ECLIPSE NOTE: If you change this file, you must restart Eclipse
+ for your changes to take effect in its Checkstyle integration. -->
+<suppressions>
+    <!--
+    Logger facades delegate to logger implementations. Unfortunately we
+    cannot opt out of specific custom rules, otherwise we would only
+    opt out of 'BanLoggingImplementations'.
+    -->
+    <suppress files="[/\\]logger-.*Bridge\.java" checks="IllegalImport" />
```

Comment:
I'm fine with this suppression given this project is explicitly is dealing with loggers, but one workaround would be to use/generate fully qualified type names where used for the things we consider illegal so they aren't imported.

Reply:
That's an option, but it would impact readability of the generated code, so I opted against it.